h/util/uri.py

```diff
@@ -132,8 +132,27 @@
 
 
 def normalize(uristr):
-    """Translate the given URI into a normalized form."""
-    uristr = uristr.encode('utf-8')
+    """
+    Translate the given URI into a normalized form.
+
+    :ptype uristr: unicode
```

Comment:
This should be `:type` not `:ptype`, docs here http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists

Reply:
Ah, I keep forgetting that. Thanks!